embed.h

```diff
@@ -1498,7 +1498,11 @@
 #    endif
 #  endif
 #  if !(defined(PERL_DEFAULT_DO_EXEC3_IMPLEMENTATION))
+#    if TARGET_OS_IPHONE
```

Comment:
Why is this necessary? I can't imagine iPhone not having exec.

Reply:
And even if it is necessary, it should probably be wired the same way Windows/DOS/OS2's `Perl_do_exec`s are wired.

Reply:
Thank you for spotting this Leon, I have pushed changes to leave embed.h intact